src/stories/components/JsPsych/JsPsych.js

```diff
@@ -0,0 +1,240 @@
+/* eslint-disable */
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { observeStimuliCompletion } from '../../../firebase/api/gcp-utils';
+import { storeExperimentResult } from '../../../firebase/api/experiments';
+import example1 from '../../../images/example1.png';
+import example2 from '../../../images/example2.png';
+
+/**
+ * Component to load jsPsych's "Self Image Experiment".
+ *
+ * @component
+ * @return {object} (
+ *   <React.Fragment />
+ * )
+ */
+export const JsPsych = () => {
+  const { experimentId, participantId } = useParams(); // Parse URL params
+  const [stimuliUrls, setStimuliUrls] = useState([]);
+  const [ready, setReady] = useState(false);
+
+  useEffect(() => {
+    // Load jsPsych library core script and styling & plugins used in the experiment
+    const jsPsychLibrary = document.createElement('script');
+    const jsPsychStyleSheet = document.createElement('link');
+    const htmlKeyboardResponsePlugin = document.createElement('script');
+    const imageKeyboardResponsePlugin = document.createElement('script');
+    jsPsychLibrary.src = '../../../external_libraries/jspsych-6.3.0/jspsych.js';
```

Comment:
Potentially using the wrong path here... (discovered by Carlo)

Proper path should be: '../../../public/external_libraries/jspsych-6.3.0/jspsych.js';

With the current implementation, JsPsych runs perfectly fine for me in Chrome. Carlo was unable to get it to run with current path / updated path containing 'public'. Need to dig more into this. Please let me know if you are / unable to run experiment on your browser!

Reply:
Just tried to run this in Chrome and unfortunately it's not working for me either

Reply:
I know we said this already but, have you tried again with loading the scripts and stylesheets (do we even need the stylesheets ever at all?) in `index.html`? To me that would be the method that's least likely to fail.

Reply:
I totally agree with you Fernando. The usual route that is taken when implementing JsPsych experiments is that the scripts & stylesheet would be loaded in index.html. 

However when I try to load it via index.html it does not properly load jsPsych library.

Reply:
Yes! Thank you.